samples/caf_sensor_manager/sample.yaml

```
@@ -0,0 +1,23 @@
+sample:
+  description: Sample showing CAF sensor manager usage
+  name: CAF sensor manager sample
+common:
+  harness: console
+  integration_platforms:
+    - nrf52840dk_nrf52840
+    - nrf52dk_nrf52832
+    - qemu_cortex_m3
+
+  platform_exclude: native_posix qemu_x86
+  harness_config:
+    type: multi_line
+    ordered: true
+    regex:
+     - "sensor_sim_ctrl state:READY"
+     - "accel_sim_xyz state:ACTIVE"
+     - "Send sensor buffer desc: accel_sim_xyz"
+     - "sensor_data_aggregator_release_buffer_event"
+
+tests:
+  samples.caf_sensor_manager:
```

```suggestion
  sample.caf_sensor_manager:
```
we are unifying the naming conventions #7290